doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -332,6 +332,6 @@ cJSON
 Documentation
 =============
 
-* |no_changes_yet_note|
+* Updated :ref:`ug_matter_gs_adding_cluster` documentation with new code snippets to align it with the source code of refactored Matter template sample.
```

Comment:
What about adding entry about PICS? In the Matter section maybe?
```suggestion
* Updated :ref:`ug_matter_gs_adding_cluster` documentation with new code snippets to align it with the source code of the refactored Matter template sample.
```

Reply:
I updated the release notes information to inform about certification data for Matter WS, Matter 1.0 release and production support for Matter over Thread.